src/modules/modals/EditContext.vue

```diff
@@ -158,7 +164,25 @@ export default {
 			this.errorTitle = false
 			this.icon.name = 'equalizer'
 			this.description = ''
-			this.resources = context ? [...this.getContextResources(context)] : []
+			this.resources = context ? this.getContextResources(context) : []
+			this.receivers = context ? this.getContextReceivers(context) : []
+		},
+		getContextReceivers(context) {
+			let sharing = Object.values(context.sharing)
+			sharing = sharing.filter((share) => getCurrentUser().uid !== share.receiver)
+			const receivers = sharing.map((share) => {
+				return {
+					user: share.receiver,
+					displayName: share.receiver,
```

Comment:
To address https://github.com/nextcloud/tables/pull/966#issuecomment-2066241589,  it seems like we need to make a separate request to fetch the actual display name.

Reply:
does that come from the regular sharing endpoint? It makes sense to return their displayname along this information.

Reply:
Alright, I see. This one's on me.

Reply:
@enjeck fixed in latest commit

![Screenshot_20240422_163654](https://github.com/nextcloud/tables/assets/2184312/31519121-11bc-4109-af58-e6aee55ec01e)

Reply:
That was fast!  :zap: :zap:  Thanks!